app/assets/stylesheets/utilities.scss

```diff
@@ -1,41 +0,0 @@
-.push-down {
-  margin-bottom: 1rem;
-}
-
-.push-down-2x {
-  margin-bottom: 2rem;
-}
-
-.push-down-3x {
-  margin-bottom: 3rem !important;
-
-  @include media-breakpoint-down(sm) {
-    margin-bottom: 1rem;
-  }
-}
-
-.full-width {
-  width: 100%;
-}
-
-.push-container-down {
-  padding-bottom: 80px;
-}
```

Comment:
Not directly related to Bootstrap, but while converting the breakpoint noticed this file was barely used. Converted the last couple of classes and tidied it while I was here